src/transformers/pipelines/table_question_answering.py

```diff
@@ -276,7 +350,7 @@ def postprocess(self, model_outputs):
         outputs = model_outputs["outputs"]
         if self.aggregate:
             logits, logits_agg = outputs[:2]
-            predictions = self.tokenizer.convert_logits_to_predictions(inputs, logits.detach(), logits_agg)
```

Comment:
Will this cause issues, or is `logits` always a Numpy array now?

Reply:
I am handling this inside `convert_logits_to_predictions`
f3ff16434bf22023965067c0a9f5fc10eac90b40